tests/unit/Pelagos/Entity/DatasetSubmissionTest.php

```diff
@@ -323,6 +323,25 @@ public function testDatasetFileAttributes()
         );
     }
 
+    /**
+     * Test dataset cold-storage related file attributes.
+     *
+     * @return void
+     */
+    public function testDatasetFileColdStorageAttributes()
```

Comment:
Can we change this to test function name to something like the existing ones? Like... testCanSetAndGetDatasetFileColdStorageAttributes (sorry for being nitpicky)

Reply:
No problem at all. I'll change.